package.json

```diff
@@ -50,7 +50,12 @@
       "plugin:prettier/recommended"
     ],
     "rules": {
-      "prettier/prettier": "warn",
+      "prettier/prettier": [
+        "warn",
+        {
+          "endOfLine": "auto"
```

Comment:
Most git repos only end lines with a line feed - is `auto` necessary for your dev environment? Is there any way we could get away with just line feed?

Reply:
I'll have to look into the settings of my IDE, the only way I could get it to stop was to set its line endings to LF but at the time since I had never tried, I only saw a way to do it per file. There is probably a way to do it per project.